code/modules/reagents/reagent_containers.dm

```diff
@@ -59,6 +59,19 @@
 				balloon_alert(user, "Transferring [amount_per_transfer_from_this]u")
 				return
 
+/obj/item/reagent_containers/attack_self_secondary(mob/user)
+	if(possible_transfer_amounts.len)
+		var/i=0
+		for(var/A in possible_transfer_amounts)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names